proto/toit/api/hardware.proto

```diff
@@ -27,6 +28,19 @@ message ClaimResponse {
   bytes organization_id = 4;
 }
 
+message ReplaceRequest {
```

Comment:
Not clear what's being replace. It should also take a `current` so we ensure we try the right thing